krux_boto/boto.py

```diff
@@ -282,7 +280,7 @@ def __init__(
 
                 # this way we can tell what credentials are being used,
                 # without dumping the whole secret into the logs
-                self._logger.debug('Setting boto credential %s to %s', env_var, BaseBoto._hide_value(val))
+                self._logger.debug('Setting boto credential %s', env_var)
```

Comment:
Now in full debug, instead of giving a partially redacted credential we dump the complete credential.  This is exactly the opposite of the stated objective of the ticket.